CharlieBackend.Api/Controllers/StudentsController.cs

```diff
@@ -158,5 +155,19 @@ public async Task<ActionResult> DisableStudent(long id)
 
             return disabledStudentModel.ToActionResult();
         }
+
+        /// <summary>
+        /// Gets filtered list of lessons for student
+        /// </summary>
+        /// <response code="200">Returned filtered list of lessons for student </response>
+        [SwaggerResponse(200, type: typeof(IList<LessonDto>))]
+        [Authorize(Roles = "Admin, Mentor, Secretary, Student")]
```

Comment:
```suggestion
        [Authorize(Roles = "Student")]
```
Only student should be allowed to access this endpoint because you are using student's account id to obtain `Student` entity object.

Then there's this problem:
![asdasd](https://user-images.githubusercontent.com/38729881/109156427-81f50a00-7779-11eb-9c97-550a447fa78d.png)

Reply:
ok, got it and fixed